cypress/e2e/tests/login.cy.ts

```diff
@@ -12,8 +12,5 @@ describe("login page", () => {
         loginActions.enterUsername("Admin");
         loginActions.enterPassword("admin123");
         loginActions.clickLogin();
-        cy.get(".oxd-topbar-header-breadcrumb > .oxd-text").should(
-            "be.visible"
-        );
     });
```

Comment:
its good point. how about to updated other similar test cases as well? which also contains similar job?

Reply:
something something